assembl/static2/js/app/components/search.jsx

```diff
@@ -130,26 +131,40 @@ const ImageType = (props) => {
   return <img className={props.className} src={`/static2/img/icon-${props.type}.svg`} alt="" />;
 };
 
+const getFieldAnyLang = (source, prop, locale) => {
+  let result;
+  if (!source) {
+    // Why does this happen?
+    return result;
+  }
+  if (source[`${prop}_${locale}`]) {
+    return source[`${prop}_${locale}`];
+  }
+  elasticsearchLangIndexes.forEach((lang) => {
+    const key = `${prop}_${lang}`;
+    if (source[key]) {
+      result = source[key];
+    }
+  });
+  if (result) {
+    return result;
+  }
```

Comment:
This condition should be inside the forEach, no? So it can return the content of the first lang in elasticsearchLangIndexes that is not empty, not the last.

Reply:
Ah, we can't do a return in a forEach, neither a break. Maybe we should just add 'en' at the end of the "elasticsearch_lang_indexes" list in this case so that English is the default?

Reply:
forEach does not return, it's not a "_.any". After the interface locale is not found, it's a random locale anyway.

Reply:
crossed messages. Now, putting en at the end is hackish, so I'll ensure we take the first, at least.

Reply:
If you wanted to keep this functional, you could have used a fold/reduce function. Result could have been the accumulator. Just a thought...

Reply:
reduce could work. do you want me to correct?